packages/dds/tree/src/simple-tree/api/typesUnsafe.ts

```diff
@@ -51,388 +51,398 @@ import type { SimpleArrayNodeSchema, SimpleMapNodeSchema } from "../simpleSchema
 export type Unenforced<_DesiredExtendsConstraint> = unknown;
 
 /**
- * {@link Unenforced} version of `ObjectFromSchemaRecord`.
- * @remarks
- * Do not use this type directly: it's only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
+ * A collection of {@link Unenforced} types that are used in the implementation of recursive schema.
+ * These are all `@system` types, and thus should not be used directly.
```

Comment:
Nit: consumers aren't likely to be aware of our `@system` tagging explicitly. Might be better to just state that the types are internal implementation details and shouldn't be referenced directly.